payments-core/src/test/java/com/stripe/android/networking/StripeApiRepositoryTest.kt

```diff
@@ -1594,14 +1594,19 @@ internal class StripeApiRepositoryTest {
                 .thenReturn(stripeResponse)
 
             val email = "email@example.com"
+            val cookie = "cookie1"
```

Comment:
I am curious if there could be an end to end test here to make sure that the encrypted cookie is cleared/deleted at the appropriate times?  For example if the user logs out, should the cookie be cleared?

Reply:
Logging out is not implemented yet. There will be unit tests for that. End to end tests for Link are a bigger lift, I'll look into that after the whole flow is implemented, especially since things are still changing.